### PR TITLE
Update text for close tab section

### LINKED ISSUE #6960: Update text for close tab settings
## Test Plan
- text's for close tab section should match this:
image

(NOTE: this task is a sub-task of #2137)

### PR DESCRIPTION
**Pull request by NejcZdovc**
## Test Plan
text's for close tab section should be the same as image attached to the original issue

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

Resolves #6960

## Auditors
@bradleyrichter

### CODE REVIEW

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 1]
```diff
@@ -131,10 +131,10 @@ startsWith=Brave starts with
 startsWithOptionLastTime=My windows / tabs from last time
 startsWithOptionHomePage=Home page
 startsWithOptionNewTabPage=a new tab
-tabCloseActionLastActive=Last active tab
-tabCloseActionNext=Next tab
-tabCloseActionFirst=First tab
-tabCloseActionParent=Parent tab
+tabCloseActionLastActive=Select the last viewed tab
+tabCloseActionNext=Select the next tab
+tabCloseActionFirst=Select the first tab
```
Comment (bradleyrichter) [inline app/extensions/brave/locales/en-US/preferences.properties]: "Select the first tab" should be removed.

File: app/extensions/brave/locales/en-US/preferences.properties [hunk 2]
```diff
@@ -131,10 +131,10 @@ startsWith=Brave starts with
...
+tabCloseActionParent=Select it's parent tab
```
Comment (Liunkae) [inline app/extensions/brave/locales/en-US/preferences.properties]: @NejcZdovc: "Select it's parent tab" should be "Select its parent tab".

Comment (bradleyrichter) [inline app/extensions/brave/locales/en-US/preferences.properties]: whoops! good catch.

Comment (NejcZdovc) [inline app/extensions/brave/locales/en-US/preferences.properties]: Fixed

File: app/extensions/brave/locales/en-US/preferences.properties (hunk 1 above)
Comment (NejcZdovc) [inline app/extensions/brave/locales/en-US/preferences.properties]: Removed

**Comment by NejcZdovc** [comment]
@bradleyrichter @Liunkae updated, please review it again

**Comment by Liunkae** [comment]
Actually, please squash the commits first. Then we can merge. 😉

**Comment by NejcZdovc** [comment]
@Liunkae rebase done

**Comment by bsclifton** [comment]
[Most tests passed with CI](https://travis-ci.org/brave/browser-laptop/builds/198955414); I re-ran each of the failed ones on my machine and they are passing.  Merging!

**Pull request merged**